source/partials/hsts.md

```diff
@@ -2,6 +2,8 @@
 
 Before adjusting `enforce_https`, review and understand the configuration options and all considerations to avoid unintended consequences.
 
+If you decide to use a plugin or module to set your HSTS header, you _must_ disable `enforce_https` in `pantheon.yml` to avoid duplicate headers, which creates an invalid policy.
```

Comment:
which creates the invalid policy?

1. disabling `enforce_https`
2. duplicate headers

If it's 2, can I change this to this or similar?

> If you decide to use a plugin or module to set your HSTS header, you _must_ disable `enforce_https` in `pantheon.yml` to avoid an invalid policy as the result of duplicate headers.